### ISSUE TITLE
Hello, How to remove the automate bridge @_bifrost_bot:aria-net.org fully?

### ISSUE THREAD

**Issue by CiangCing14** [matrix-org/matrix-bifrost#363]
Hello, How to remove the automate bridge [@_bifrost_bot:aria-net.org](https://matrix.to/#/@_bifrost_bot:aria-net.org) fully?
https://archon.im/arianet/matrix-homeserver/services/
Tried !bifrost leave , but failed. Now there is some "[@_bifrost_USERNAME=40DOMAIN_NAME:aria-net.org](https://matrix.to/#/@_bifrost_USERNAME=40DOMAIN_NAME:aria-net.org)" rest there. and join in, leave the room. their join and leave logs filled the room message history.
